tsdb/tsdbblockutil.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tsdb
+
+import (
+	"context"
+	"fmt"
+	"github.com/go-kit/kit/log"
+	tsdb_labels "github.com/prometheus/prometheus/tsdb/labels"
+	"os"
+	"path/filepath"
+)
+
+// Default duration of a block in milliseconds - 2h.
+const (
+	DefaultBlockDuration = int64(2 * 60 * 60 * 1000)
```

Comment:
If we are introducing this const lets set it in the main db.go file and use it in `ExponentialBlockRanges`

Reply:
You mean you change this line: https://github.com/prometheus/prometheus/blob/de0a772b8e7d27dc744810a1a693d97be027049a/tsdb/db.go#L51

to be `BlockRanges:     DefaultBlockDuration`?

Reply:
Uh, sorry, I misunderstood. You want to use it like so: `ExponentialBlockRange(DefaultBlockDuration, 3, 5)`